packages/server/lib/browsers/webkit-automation.ts

```diff
@@ -30,22 +30,25 @@ function convertSameSiteExtensionToCypress (str: CyCookie['sameSite']): 'None' |
   return str ? extensionMap[str] : undefined
 }
 
-const normalizeGetCookieProps = (cookie: any): CyCookie => {
-  if (cookie.expires === -1) {
-    delete cookie.expires
+const normalizeGetCookieProps = ({ name, value, domain, path, secure, httpOnly, sameSite, expires }: playwright.Cookie): CyCookie => {
+  const cyCookie: CyCookie = {
+    name,
+    value,
+    domain,
+    path,
+    secure,
+    httpOnly,
+    // Use expirationDate instead of expires 🤷‍♀️
+    ...expires !== -1 ? { expirationDate: expires } : {},
+  } as CyCookie
```

Comment:
You defined this object type as CyCookie above with `cyCookie: CyCookie` so no need to cast it here
```suggestion
  }
```